addons/web/static/src/js/fields/relational_fields.js

```diff
@@ -2304,6 +2304,76 @@ var FieldRadio = FieldSelection.extend({
     },
 });
 
+
+var FieldSelectionBadge = FieldSelection.extend({
+    template: null,
+    className: 'o_field_selection_badge',
+    tagName: 'span',
+    specialData: "_fetchSpecialMany2ones",
+    events: _.extend({}, AbstractField.prototype.events, {
+        'click span.o_selection_badge': '_onBadgeClicked',
+    }),
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * @private
+     * @override
+     */
+    _renderEdit: function () {
+        var self = this;
```

Comment:
Linting: using a linter should show this variable is not used anywhere. Please check the whole diff for unnecessary variables.

Reply:
yes, definitely yes!  if only github would show unused variables also.  I wish we could use a code formatter to totally remove that kind of issue.